### ISSUE TITLE
Is it possible to build Windows 10 apps on Windows 8.1?

### ISSUE THREAD

**Issue by brody2consult** [litehelpers/Cordova-sqlite-help#12]
As asked in https://github.com/litehelpers/Cordova-sqlite-legacy-build-support/issues/12:
> Please suggest, if yes then can we use cordova sql lite plugin for data storage.
> Please share some document link if possible.
> Also please suggest if we could test the windows cordova app without having any developer account.

It seems to be possible to develop Windows 10 apps on Windows 8.1: https://msdn.microsoft.com/en-us/library/dn975273.aspx

I have never tried this myself and have no idea if this would work with Cordova and suspect it may become even more tricky when using some form of the sqlite plugin.

Also that document was written for Visual Studio 2015, maybe obsolete for Visual Studio 2017.

In short: not tested and not recommended.
